src/app/datagrid/datagrid-paging-service-demo.service.ts

```diff
@@ -25,7 +25,7 @@ export class DataGridPagingServiceDemoService extends SohoDataGridService {
     const end: number = begin + req.pagesize;
     const data: Array<any> = this.data.slice(begin, end);
     const result: any = { total: 100, data: data };
-    return Observable.of(result);
+    return observableOf(result);
```

Comment:
I see above you've used just `of` and here you're using `observableOf`. Why the difference?